halpybot/packages/board/board.py

```diff
@@ -111,6 +104,11 @@ def open_rescue_id(self) -> int:
             return self._open_rescue_id
         return next_id
 
+    @property
+    def by_id(self) -> dict:
```

Comment:
[non-blocking] `dict` is an insufficient type annotation.

use parametrized form `typing.Dict[KT, VT]`